app/javascript/controllers/feedback_controller.js

```diff
@@ -0,0 +1,51 @@
+import { Controller } from "stimulus"
+import Rails from '@rails/ujs'
+
+export default class extends Controller {
+
+  static targets = [ "reply", "container" ]
+
+  update() {
+    var feedbackId = event.target.dataset.value
+    let data = new FormData()
+    data.append("id", feedbackId)
+    var url = "/feedback/" + feedbackId
+    Rails.ajax({
+      url,
+      type: "PATCH",
+      data
+    })
+  }
+
+  showCollapsable() {
+    var collapsable = event.target.dataset.id
+    var id = "#" + collapsable
+    if($(id).hasClass('show'))
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/birs-math/proposals/pullRequest?prid=7697799)
```suggestion
    if($(id).hasClass('show'))
```